client/my-sites/media-library/content.scss

```diff
@@ -22,3 +22,17 @@
 .media-library__scale-range .range__content.is-max {
 	right: -28px;
 }
+
+.media-library__connect-message {
+	max-width: 480px;
+	padding: 36px;
+	margin: 0 auto;
+	text-align: center;
+}
+
+.media-library__source-picker {
```

Comment:
It doesn't seem to be - @jasmussen is this CSS needed? I've tried it without and it looks the same.